data_pipeline/concepts.py

```diff
@@ -0,0 +1,87 @@
+from random import randrange
+from typing import List, Dict
+
+import penman
+from penman.graph import Triple
+from penman.surface import alignments
+
+
+def get_concept_ids(g: penman.Graph) -> List[int]:
+  """
+  Constructs a list of concept ids.
+
+  Args:
+    g: amr graph (in penman library representation).
+  Returns:
+    A list of the triple ids (positions in the original triples list) for the
+    triples associated with concepts (that have a concept on the 3rd position). 
+  """
+  triples = g.triples
+  variables = [t[0] for t in triples]
+  variables = list(set(variables))
+  filtered_triples_ids = []
+  for triple_id in range(len(triples)):
+    t = triples[triple_id]
+    src = t[0]
```

Comment:
This is not used. Is there a reason you declared it?

Reply:
Removed now